ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_activations.py

```diff
@@ -226,6 +226,34 @@ def test_silu(*, dtype_and_x, test_flags, backend_fw, fn_name, on_device):
     )
 
 
+# softshrink
+@handle_test(
+    fn_tree="functional.ivy.experimental.softshrink",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        large_abs_safety_factor=8,
+        small_abs_safety_factor=8,
+        safety_factor_scale="log",
```

Comment:
Can you use `valid` instead? And add supported dtypes to the relevant functions

Reply:
i made the changes